### PR TITLE
Homepage and videospage

### PR DESCRIPTION
**Pull request by ParthParolekar**
Home Page and videos page added with authenctication

### CODE REVIEW

File: src/components/Navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  display: block;
+  cursor: pointer;
+  font-size: 2rem;
+}
+
+.nav-links {
+  position: absolute;
+  left: 0;
+  right: 0;
+  /* top: -180px; */
```
Comment (MayankKumar10) [inline src/components/Navbar/navbar.css]: remove this line

File: src/components/Banner/banner.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 10px;
+}
+
+.banner-heading {
+  position: absolute;
+  top: 50%;
```
Comment (MayankKumar10) [inline src/components/Banner/banner.css:30]: Suggestion: Try to avoid using large values in the top or margin it will affect in responsiveness

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions everything else looks good you can merge it.

File: src/components/Banner/banner.css (hunk 2 above)
Comment (ParthParolekar) [reply src/components/Banner/banner.css:30]: I used percentage for this exact reason so that no matter the screen size, It will always work

**Pull request merged**